### PR TITLE
Fix: project tribute error handling 2

### PR DESCRIPTION
**Pull request by cptnanon**
Show warning if the wallet gets disconnected

### CODE REVIEW

File: frontend/src/pages/Projects/ProjectTribute.tsx [hunk 1]
```diff
@@ -115,14 +118,24 @@ export default function ProjectTribute() {
...
+          <Link onClick={openConnectModal} color={'#0000EE'} sx={{ cursor: 'pointer' }}>
+            here
+          </Link>{' '}
+          to connect again.
+        </Alert>
+      )}
+
       {!isManager && (
         <Alert severity="error" sx={{ overflow: 'hidden', wordBreak: 'break-word' }}>
           You are not the manager of this project. Your wallet account "{userAddress}" does not match the manager
```
Comment (ivelin) [inline frontend/src/pages/Projects/ProjectTribute.tsx:133]: @cptnanon do we need to add `&& !isDisconnected` to this Alert's conditional rendering logic?

Comment (cptnanon) [reply frontend/src/pages/Projects/ProjectTribute.tsx:133]: Are you suggesting to use the `userAddress` as the condition?

**Comment by cptnanon** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cptnanon** [comment]
recheck

File: frontend/src/pages/Projects/ProjectTribute.tsx (hunk 1 above)
Comment (ivelin) [reply frontend/src/pages/Projects/ProjectTribute.tsx:133]: Either that or test with isDisconnected. Line 131 would become:
```typescript
{!isManager && !isDisconnected (
```

That should prevent rendering empty userAddress, which makes the message confusing

> Your wallet account "" does not match the manager

See these screenshots:
https://github.com/SporosDAO/sweat-token/pull/142#issuecomment-1256254383

**Comment by ivelin** [comment]
> I have read the CLA Document and I hereby sign the CLA

It seems like you used a different github id ([CPTN021](https://github.com/CPTN021)) to commit to this PR, which is why this CLA is not passing.

It should pass if you login and comment here as  CPTN021.

**Review by ivelin — APPROVED** [review]
@cptnanon awesome work. Thank you!
Merging.

**Pull request merged**